frontend/src/components/Search/SearchForm/SearchForm.tsx

```diff
@@ -308,7 +296,11 @@ export const Search: React.FC<{
 	if (showOperators) {
 		const operators =
 			keyMatch.type === 'Numeric'
-				? [...NUMERIC_OPERATORS, ...EXISTS_OPERATORS]
+				? [
+						...BOOLEAN_OPERATORS,
```

Comment:
Not directly related, but added boolean operators since they should be used with numbers as well